inc/frame.h

```diff
@@ -0,0 +1,22 @@
+#ifndef _FRAME_H
+#define _FRAME_H
+
+/*
+   |   0    |    1   | 2 - 17 |
+   | opcode | status |  code  |
+*/
+
+enum OPCODE {
+    REQ_PAIR,
+    PAIRED,
+    REQ_TURN,
+    REQ_STATUS,
+    RES,
+    REQ_PLACE
+};
+
+#define STATUS_INV -1
+#define STATUS_OK 1
+#define STATUS_FAILED 0
```

Comment:
Are we going to use the lower 8 bits as the `status` code?

Reply:
Yes, the lower 8 bits will be used as `status` code.

The status codes you provided seem great, I'll change the code to this version

Reply:
Great, thanks!